src/databases/MFIXCDF/avtMFIXCDFFileFormat.C

```diff
@@ -1449,14 +1448,8 @@ avtMFIXCDFFileFormat::GetVectorVar(int domain, const char *varname)
         nzvals= (widths[2]+3);
     }
 
-    // This trick with auto_ptr makes xvec, yvec and zvec get deleted
-    // when they go out of scope.
-    auto_ptr< vector<float> > xvec(new vector<float>(totZones));
-    float* xdata= &(*xvec)[0];
-    auto_ptr< vector<float> > yvec(new vector<float>(totZones));
-    float* ydata= &(*yvec)[0];
-    auto_ptr< vector<float> > zvec(new vector<float>(totZones));
-    float* zdata= &(*zvec)[0];
+    vector<float> xvec(totZones), yvec(totZones), zvec(totZones);
+    float *xdata = &xvec[0], *ydata = &yvec[0], *zdata = &zvec[0];
```

Comment:
Not sure if it is specified in our style docs, but this is a little confusion to read. I believe it would be more clear if each was on its own line. The above (1451) is not as confusing, but could also benefit from being on their own lines.

Reply:
My view...vertical space is a premium. Horizontal space less so. This is why whenever I can, I keep lobbying for 132 col source files. The 80 column limit is a hold-over from the 1960s. But, will change as requested.